src/main/java/org/opensearch/ad/AnomalyDetectorExtension.java

```diff
@@ -39,7 +44,13 @@ public ExtensionSettings getExtensionSettings() {
 
     @Override
     public List<ExtensionRestHandler> getExtensionRestHandlers() {
-        return List.of(new RestCreateDetectorAction());
+        List<ExtensionRestHandler> handler = null;
+        try {
+            handler = List.of(new RestCreateDetectorAction());
```

Comment:
We don't need a catch block for this object instantiation.  The constructor is a no-op.